roles/mig_controller_prereqs/tasks/operator_upstream_olm.yml

```diff
@@ -1,3 +1,16 @@
+- name: "Obtain latest konveyor channel"
```

Comment:
Only one question here as I am unsure. Is the mig-operator repository always available at {{ playbook_dir }}/mig-operator location?

Reply:
Actually it should be {{ mig_operator_location }} which controls where the git clone task dumps operator, fixed.